packages/core/src/test/shared/telemetry/spans.test.ts

```diff
@@ -685,6 +685,44 @@ describe('TelemetryTracer', function () {
                 (e) => e === arbitraryError
             )
         })
+
+        const customWithTelemetryContext = withTelemetryContextFactory({ class: 'TestCustomContext', emit: true })
```

Comment:
Modules can define their own predefined fields. That's more explicit, and avoids this extra layer (we already have too many layers). And avoids these extra tests.

Reply:
Makes sense, removed